src/components/STooltip.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <VTooltip
+    v-bind="$attrs"
+    dark
+    open-on-click
+    :open-on-hover="false"
+    :content-class="tooltipStyle"
+    bottom
+    max-width="320px"
+    open-delay="500"
+    nudge-left="10px"
+    nudge-bottom="8px"
+    class="pa-3"
+  >
+    <template v-slot:activator="{ on }">
+      <SBtn
+        v-show="icon"
+        :height="size"
+        :width="size"
+        icon
+        v-on="on"
+      >
+        <VIcon :color="color" :size="size">
+          {{ icons.mdiHelpCircleOutline }}
+        </VIcon>
+      </SBtn>
+      <SBtn
+        v-show="!icon"
+        link-medium
+        v-on="on"
+      >
+        {{ text }}
+      </SBtn>
+    </template>
+    <slot />
+  </VTooltip>
+</template>
+<script lang="ts">
+import { Component, Prop, Vue } from 'nuxt-property-decorator'
+import { mdiHelpCircleOutline } from '@mdi/js'
+import SBtn from '../../src/components/SBtn.vue'
```

Comment:
i`m not sure if this will work e.g. path like this. Probably you need to start with './

Reply:
updated